warehouse/query-core/src/main/java/datawave/query/index/lookup/RangeStream.java

```diff
@@ -544,7 +544,7 @@ public ScannerStream visit(ASTEQNode node, Object data) {
                     return ScannerStream.unindexed(node);
                 }
             } catch (TableNotFoundException e) {
-                log.error(e);
+                log.error(String.valueOf(e));
```

Comment:
I think these should just be `log.error(e)`. Is there no method for that? We want to log the full stacktrace. If we convert to a String, then we lose that.

Reply:
I see. So we need some kind of message. We might be able to get away with `log.error("", e);`.